### DISCUSSION TITLE
Individual Email Results from Marketing Cloud not working

### DISCUSSION THREAD

**Discussion by stefanmelstad23** [deejay-hub/timeline-lwc#251] [Q&A] [answered]
I'm unable to get Individual Email Results from Marketing Cloud to work on a Person Account timeline. This custom object (et4ae5__IndividualEmailResult__c) is part of the Marketing Cloud managed package. It has a lookup relationship to the contact part of a Person Account.

Is there something I'm doing wrong or is this not supported?

![image](https://github.com/deejay-hub/timeline-lwc/assets/53267023/9810445e-82c1-4754-aa39-790f867dcbe4)

**Comment by deejay-hub** [comment, marked as answer]
The relationship name looks like it should be **et4ae5__IndividualEmailResults__pr**.   Note the addition of the _p_.

See [Dynamic SOQL and Salesforce](https://medium.com/@dave-norris/dynamic-soql-person-accounts-2cd43a552149) for an explanation of how Salesforce differentiates between relationships for Person Accounts.

Let me know if this helps. Otherwise I'll have to install the package.

**Reply by deejay-hub** [reply]
@stefanmelstad23 any update? We'll leave this discussion open for another few days then assume our suggestion has helped.
